plugins/GSdx/GSLinuxDialog.cpp

```diff
@@ -351,7 +351,7 @@ void populate_hack_table(GtkWidget* hack_table)
 	GtkWidget* hack_tco_entry      = CreateTextBox("UserHacks_TCOffset");
 	GtkWidget* align_sprite_check  = CreateCheckBox("Align sprite hack", "UserHacks_align_sprite_X");
 	GtkWidget* preload_gs_check    = CreateCheckBox("Preload Frame", "preload_frame_with_gs_data");
-	GtkWidget* hack_unsafe_fbmask  = CreateCheckBox("Fast Accurate Blending", "UserHacks_unsafe_fbmask");
+	GtkWidget* hack_safe_fbmask    = CreateCheckBox("Fast Accurate Blending", "UserHacks_safe_fbmask");
```

Comment:
would be better changing it to ``Safe accurate blending``. the checkbox text  for windows doesn't use a uppercase letter for ``Accurate`` and ``Blending``.